src/components/Accordion/Accordion.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { shallow } from 'enzyme';
+import Accordion from './Accordion';
+import { accordionItemsData } from './AccordionData';
+
+describe('<Accordion />', () => {
+  let wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<Accordion data={accordionItemsData}/>);
+  });
+
+  it('should match snapshot <Accordion />', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should change hide state property to "false" when you click on Accordion item for the first time, for the second time to true',
```

Comment:
You can shorten this line. No need to specify what Accordion item does. It's clear enough from test context